notebooks/community/feature_store/mobile_gaming_feature_store.ipynb

```diff
@@ -0,0 +1,2274 @@
+{
```

Comment:
<p>I think readers will get lost here, will be good to explain what's in the <code>read instances</code> , and what they mean. i.e. the read instances is a list of entity id / timestamp pairs, where the entity id is the customer id? and timestamp indicates we want to read the latest information about that customer before this indicated time.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='94'/>

Reply:
<p>Done. </p>